lms/static/styles/components/_LMSFilePicker.scss

```diff
@@ -1,8 +1,8 @@
 .LMSFilePicker__dialog {
-  display: flex;
-  flex-direction: column;
-  max-height: 100vh;
-  max-width: 500px;
-  width: 80vw;
-  height: 400px;
+  /* Center the dialog over the content. */
```

Comment:
I'm not sure these settings are necessary at all anymore. I think the default width of 75% looks very reasonable already. If the content exceeds the parent height then it turns into a scroll so I don't think there needs to be a max height setting on here particularly to the whole viewport as the height would not exceed the parent element. Rather than setting a particular height, allowing the default height to auto adjust based on the size of the content also seems preferable to me (although this does cause the content to jump up when the error details are expanded and the content was smaller than the size of the parent due to the margin:auto where the content is centered within the parent element). I don't see that the flex settings add any value here either. For these reasons I've removed these two classes entirely. Let me know if you think otherwise!